umap/static/umap/js/modules/importer.js

```diff
@@ -234,9 +234,17 @@ export default class Importer {
   }
 
   submit() {
-    if (this.format === 'umap') this.full()
-    else if (!this.url) this.copy()
-    else if (this.action) this[this.action]()
+    let hasErrors = false
```

Comment:
I'm not fully convinced by that approach but I couldn't find a clever way to only display the successful message in case there is no error during the potential imports.